core/src/main/java/org/arquillian/smart/testing/logger/Logger.java

```diff
@@ -3,8 +3,7 @@
 public interface Logger {
 
     /**
-     * Will format the given message with the given arguments and prints it on standard output with the prefix:
-     * "[INFO] [Smart Testing Extension]"
+     * Will format the given message with the given arguments and print it on standard output.
```

Comment:
> ...and print it on standard output.

On the interface level, we don't know where the information will be printed. I would just say that the message will be logged at the info level. analogically for the following methods.